detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UseIfInsteadOfWhen.kt

```diff
@@ -34,9 +37,15 @@ class UseIfInsteadOfWhen(config: Config = Config.empty) : Rule(config) {
         Debt.FIVE_MINS
     )
 
+    @Configuration("ignores when variable declaration is used in subject")
```

Comment:
```suggestion
    @Configuration("ignores when statements with a variable declaration used in the subject")
```

as `when` is ambigous here